chat-plugins/scavengers.js

```diff
@@ -144,13 +197,86 @@ exports.commands = {
 		scavengers.answers = null;
 		scavengers.participants = {};
 		scavengers.finished = [];
+		scavengers.updatinglb = false;
+		scavengers.numblitzes = 0;
 		if (scavengers.result) {
 			Rooms('scavengers').addRaw('<div class="broadcast-blue"><strong>' + scavengers.result + '</strong></div>');
```

Comment:
Um, if `scavengers.result` is sanitized HTML, it should be named `scavengers.resultHTML`, and if not, it should be sanitized.